compiler/util/misc.cpp

```diff
@@ -35,6 +35,9 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 
+
+static const char* help_url = "http://chapel.cray.com";
+
```

Comment:
I suggest stting help_url to this:
  http://chapel.cray.com/getinvolved.html

Getting to that page from the initial page requires a significant effort - let's not impose it on the users.